content/post/invitation-to-test.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Invitation to test : IR hosting service"
+date: 2020-09-28T17:00:00+02:00
+draft: false
+tags: ["iraas", "repository", "shared portal", "dedicated portal"]
+---
+
+One of the goals of the [SONAR project](https://sonar.ch/) is to set up a hosting service for institutional repositories (IRaaS - Institutional Repository as a Service). [Two variants](https://sonar.ch/iraas/) of this service are proposed: a shared portal and dedicated portals. In anticipation of its first production launch, this service is currently providing a public test instance at the address [sonar.test.rero.ch](https://sonar.test.rero.ch).
```

Comment:
In anticipation of its first production launch, this service is currently providing
to 
In anticipation of its first production launch, the service is currently providing

Reply:
OK.